src/Sprite/ArtFile.h

```diff
@@ -14,10 +15,13 @@ namespace Stream {
 	class Writer;
 }
 
+// ArtFile stores sprites in 1 and 8 bit indexed bitmaps. 2 bit bitmaps represent shadows.
+// ArtFile may be packaged within a vol file and still register with Outpost 2.
 struct ArtFile
 {
 public:
-	std::vector<Palette> palettes;
+	// Palettes within ArtFile are stored in 8 bits of memory regardless of actual palette size (1 or 8 bit)
```

Comment:
Is this comment still accurate? Or are palettes now sized according to use? Or are some palettes used by both 1 bpp and 8 bpp bitmaps?

Reply:
Yes, currently the ArtFile class represents the file memory footprint and allocates space for 8bpp palettes for all sprites. I'm not advocating for this model over others besides it seemed easiest to implement at the time.